control/statefbk.py

```diff
@@ -112,18 +112,32 @@ def place(A, B, p):
     return K
 
 
-def place_varga(A, B, p):
+def place_varga(A, B, p, DICO='C', alpha=None):
     """Place closed loop eigenvalues
-    K = place_varga(A, B, p)
+    K = place_varga(A, B, p, DICO='C', alpha=None)
 
-    Parameters
+    Required Parameters
     ----------
     A : 2-d array
         Dynamics matrix
     B : 2-d array
         Input matrix
     p : 1-d list
         Desired eigenvalue locations
+
+    Optional Parameters
+    ---------------
+    DICO : 'C' for continuous time pole placement or 'D' for discrete time.
```

Comment:
The use of `DICO` as a parameter seems odd since it is not used in other functions.  A couple of possibilities:

* `_rss_generate` (internal function) uses `type`
* `gram` uses `dico` (lower case) as an internal variable
* We use use `isdtime` and `isctime` as functions to check for discrete and continuous time.  We could use `isdtime=False` as the option.

Reply:
I am also in favour of isdtime, or maybe simply "dtime". The C / D character is a really Fortran-like approach.

Reply:
Agree that `dtime=False` would be a better choice for the option (and default value).